### DISCUSSION TITLE
Historias de usuario

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [workflix/workflix-full#2] [General]
Creo esta discusión para que vayamos proponiendo Historias de Usuario.

#US01 Como usuario cliente, en la página web, me gustaría que en la página inicial se muestre la opción para descargar la aplicación en el celular.

**Comment by yan0106** [comment]
#US02 Como usuario cliente, en la app mobile, me gustaría tener un botón para que me redireccione a la página web institucional.

**Comment by yan0106** [comment]
Historias de usuarios que discutimos en la daily:

#US01: Como usuario cliente, en la página web, me gustaría que en la página inicial se muestre la opción para descargar la aplicación en el celular. 
#US02: Como usuario cliente, me gustaría poder visualizar en la web y en la app los mismos profesionales para elegir cuál contratar.
#US03 Como usuario cliente, me gustaría que la web y la app muestren el mismo diseño para que sea consistente.
#US04 Como usuario cliente, en la app mobile, me gustaría tener un botón que me redireccione a la página web institucional para obtener más información.
#US05 Como usuario visitante me gustaría poder visualizar los profesionales mejor calificados de la app mobile para elegir el profesional adecuado.

**Comment by NataliaAlvarezIspc** [comment]
Nuevas US:

#US06 Como usuario registrado me gustaría poder recuperar mi contraseña de ingreso en el caso de olvidarla para poder ingresar nuevamente al sitio o a la aplicación móvil
#US07 Como usuario visitante, en la aplicación móvil, me gustaría poder filtrar los profesionales para poder elegir con más facilidad según el rubro que busco.
#US08 Como usuario visitante, en la aplicación móvil, me gustaría poder contactar directamente por whatsApp a los profesionales, para que sea más sencillo contactarlos.
